apps/src/p5lab/P5Lab.js

```diff
@@ -1572,12 +1594,13 @@ export default class P5Lab {
       // feedbackImage: feedbackImageCanvas.canvas.toDataURL("image/png")
       showingSharing: !level.disableSharing && level.freePlay,
       appStrings: {
-        reinfFeedbackMsg: msg.reinfFeedbackMsg(),
+        reinfFeedbackMsg,
         sharingText: msg.shareGame()
       },
       hideXButton: true,
       saveToProjectGallery: saveToProjectGallery,
-      disableSaveToGallery: !isSignedIn
+      disableSaveToGallery: !isSignedIn,
+      disablePrinting: this.disablePrinting()
```

Comment:
in our post-HoC follow-up work, this function will be changed to `false` to disable printing in all P5Labs. being so close to HoC, we only wanted to disable printing in Spritelab/Poetry for now so we can re-evaluate this configuration setting across all other labs later. it should likely only be enabled in Artist, so this setting is probably backwards (e.g., you should have to opt in, not opt out)

Reply:
this dialog is the only place where this setting is consumed:

https://github.com/code-dot-org/code-dot-org/blob/6be244bafd471fe390edba08077acdcb0db55c9e/apps/src/templates/sharing.html.ejs#L33-L37

Reply:
Howdy - maybe I'm not understanding - it sounds like we want to only disable printing in Spritelab/Poetry, but this.disablePrinting returns false, so will it disable printing across all P5Lab?

Reply:
woops, my first comment was the opposite of what i wanted! i updated the comment, but also want to clarify here:

after HoC, we'll remove the `this.disablePrinting` function and update this configuration to `{disablePrinting: true}` for all P5Labs (and likely other labs)